addons/website/models/mixins.py

```diff
@@ -230,8 +230,6 @@ def _get_can_publish_error_message(self):
 
 
 class WebsitePublishedMultiMixin(WebsitePublishedMixin):
-
-    _name = 'website.published.multi.mixin'
```

Comment:
Can't the script just fix this? How is that not a covered case? :thinking:

Reply:
`WebsitePublishedMixin`